### PR TITLE
1384999 update server broadcast with signal r 2

### PR DESCRIPTION
**Pull request by john-par**
Fixes #1384999
Update for task to use .NET tutorial template.

### CODE REVIEW

**Comment by john-par** [comment]
@tdykstra this PR is ready for your review. Please let me know what you'd like changed.

Also, I have a question about the stock symbols that are used in [the *StockTicker.cs*](https://review.docs.microsoft.com/en-us/aspnet/signalr/overview/getting-started/tutorial-server-broadcast-with-signalr?branch=pr-en-us-10156#create-stocktickercs):

```csharp
 …
{
   new Stock { Symbol = "MSFT", Price = 30.31m },
   new Stock { Symbol = "APPL", Price = 578.18m },
   new Stock { Symbol = "GOOG", Price = 570.30m }
};
```
I know that there [fictitious companies and URLs](https://review.docs.microsoft.com/en-us/help/contribute/contribute-legal-guidelines?branch=master#fictitious-names) that can be used in articles. Does the same exist for stock symbols? I wasn't able to find any guidance on the topic.

File: aspnet/signalr/overview/getting-started/tutorial-server-broadcast-with-signalr.md [hunk 1]
```diff
@@ -1,407 +1,495 @@
...
+The application that you'll create in this tutorial simulates a stock ticker, a typical scenario for server broadcast functionality. Periodically, the server randomly updates stock prices and broadcast the updates to all connected clients. In the browser, the numbers and symbols in the **Change** and **%** columns dynamically change in response to notifications from the server. If you open additional browsers to the same URL, they all show the same data and the same changes to the data simultaneously.
 
+> [!IMPORTANT]
+> If you don't want to work through the steps of building the application, you can install the SignalR.Sample package in a new Empty ASP.NET Web Application project. If you install the NuGet package without performing the steps in this tutorial, you must follow the instructions in the *readme.txt* file. To run the package you need to add an OWIN startup class which calls the `ConfigureSignalR` method in the installed package. You will receive an error if you do not add the OWIN startup class.
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-server-broadcast-with-signalr.md]: Do these instructions and the tutorial build-from-scratch instructions both work?

Comment (john-par) [reply aspnet/signalr/overview/getting-started/tutorial-server-broadcast-with-signalr.md]: Yes, I was able to get both of these to work with VS 2017.

**Comment by tdykstra** [comment]
>Does the same exist for stock symbols? I wasn't able to find any guidance on the topic.

I don't believe there are standard fictitious stock symbols.

**Comment by john-par** [comment]
> One thing I didn't comment inline on -- there were a lot of minor wording changes that weren't done in order to comply with MVC guidelines (e.g., "such as" changed to "like") . In the future I'd prefer we not make those kinds of changes as they weren't requested, and they take time to do and to review.

Thanks for that note. I will stop making those types of changes.

**Comment by john-par** [comment]
@tdykstra all of your feedback has been incorporated.

**Pull request merged**